detection_rules/schemas/definitions.py

```diff
@@ -27,7 +27,11 @@
 VERSION_PATTERN = f'^{_version}$'
 MINOR_SEMVER = r'^\d+\.\d+$'
 BRANCH_PATTERN = f'{VERSION_PATTERN}|^master$'
-
+ELASTICSEARCH_EQL_FEATURES = {
+    "allow_runs": (Version.parse('7.16.0'), None),
+    "allow_sample": (Version.parse('8.6.0'), None),
+    "elasticsearch_validate_optional_fields": (Version.parse('7.16.0'), None)
+}
```

Comment:
I'd appreciate extra eyes on the release dates.

Reply:
Not sure if I am looking for the right thing, but it looks like documentation for `sample`[ was introduced in v.8.6](https://github.com/elastic/elasticsearch/pull/92504) so that looks good :+1:

Reply:
Yup, just wanted to validate the two other features.